kolibri/core/assets/src/views/KGrid/extra-units.css

```diff
@@ -78,7 +77,7 @@
 }
 
 .pure-u-7-7 {
-  width: 100.0%;
+  width: 100%;
```

Comment:
I don't know for sure whether this was intentional or necessary - it's based on pure.css grids and they do some fairly tricky browser compatibility stuff. I'd prefer to ignore the file from linting and keep the decimal precision as-is to be on the safe side if possible.

Reply:
Righto, can also add linting exclude comments.